root/help.md

```diff
@@ -0,0 +1,41 @@
+% DHCP-CLIENT (1) Container Image Pages
+% Karsten Hopp
+% June 13, 2017
+
+# NAME
+dhcp-client - Dynamic Host Configuration Protocol Client
+
+# DESCRIPTION
+A DHCP client container running as a root user based on Fedora 26 Boltron. Provides a means for configuring one or more network interfaces using DHCP.
+
+# USAGE
+For running in docker use:
+
+	# docker run --net=host --cap_add=net_admin dhcp-client-container
+
+If you want to run this in openshift, you need to have SCC RunAsUser set to RunAsAny. Then run:
+
+	# oc create -f openshift-template.yml
+
```

Comment:
Please add a sentence (link to this github repo) where users can obtain the template file.